.github/workflows/update_v8_snapshot_cache.yml

```diff
@@ -35,7 +37,8 @@ jobs:
     env:
       CYPRESS_BOT_APP_ID: ${{ secrets.RAM_APP }}
       BASE_BRANCH: ${{ inputs.branch || github.ref_name }}
-      GENERATE_FROM_SCRATCH: ${{ inputs.generate_from_scratch == true || github.event_name == 'schedule' }}
+      # Flex the generate from scratch option based on manual input or if we are on the weekly schedule
+      GENERATE_FROM_SCRATCH: ${{ inputs.generate_from_scratch == true || (github.event_name == 'schedule' && github.event.schedule == '0 0 * * 3') }}
```

Comment:
should this one be every day too or only on Wednesday?

Reply:
I want "from scratch" to happen weekly. And the other one to happen daily.